atc/syslog/syslog.go

```diff
@@ -94,15 +94,15 @@ func (s *Syslog) Close() error {
 }
 
 // generate custom formatter based on hostname and tag
-func getSyslogFormatter(hostname string, ts time.Time, tag string) sl.Formatter {
+func getSyslogFormatter(hostname string, ts time.Time, tag, eventID string) sl.Formatter {
 	return func(priority sl.Priority, _, _, content string) string {
 		// strip whitespaces
 		s := strings.Replace(content, "\n", " ", -1)
 		s = strings.Replace(s, "\r", " ", -1)
 		s = strings.Replace(s, "\x00", " ", -1)
 
-		msg := fmt.Sprintf("<%d>1 %s %s %s - - - %s\n",
-			priority, ts.Format(rfc5424time), hostname, tag, s)
+		msg := fmt.Sprintf("<%d>1 %s %s %s - - [meta sequenceId=\"%s\"] %s\n",
```

Comment:
`eventId` is generated per `build+eventType`, should build number and eventType be logged as well?

Reply:
"build number" actually already included in the `tag`: `<team>/<pipeline>/<job>/<build_id>/<origin_id>`.